docs/cli-existing-project.md

```diff
@@ -52,8 +52,7 @@ foundryup
 ## 3. Install Sphinx CLI
 
 First, navigate to the root of your repository.
-
-Then, install Sphinx using your preferred package manager.
+Navigate to your smart contract workspace. In a standard repo, this is the root of your project. In a monorepo, you should move to your contracts package. Then, install Sphinx using your preferred package manager.
```

Comment:
> Then, install Sphinx using your preferred package manager.

nit: Would be nice if this sentence is on a separate line from the preceding sentence because it's sort of a separate step